src/htr2hpc/train/run.py

```diff
@@ -25,8 +35,9 @@
 
 # kraken python package provides paths to the default best models for both modes
 default_model = {
-    "Segment": SEGMENTATION_DEFAULT_MODEL,
-    "Recognize": DEFAULT_MODEL,
+    "Segment": SEGMENTATION_DEFAULT_MODEL,  # this is a PosixPath
+    # this is a list of string, relative path, but file does not actually exist
+    "Recognize": DEFAULT_MODEL[0],
```

Comment:
@cmroughan have you run into this or do you have any advice about this? The kraken code has a defined variable for a default model for recognition, but it is just a name, not a path and doesn't actually seem to exist anywhere in my local installation. Do we need to require a model for fine-tuning?

Reply:
I have never interacted with a default model for transcription (unlike the `blla.mlmodel` for segmentation). When doing transcription training, we will either be training the model from scratch or we will be finetuning on a different model that the user has selected in eScriptorium. So no need to track down some sort of default transcription model in kraken!

Reply:
Thanks for explaining. I've removed the default model for recognition training and made input model optional, we can test it when della is back up after maintenance today.